cg/apps/demultiplex/demultiplex_api.py

```diff
@@ -42,6 +43,21 @@ def slurm_quality_of_service(self) -> Literal[SlurmQos.HIGH, SlurmQos.LOW]:
         """Return SLURM quality of service."""
         return SlurmQos.LOW if self.environment == "stage" else SlurmQos.HIGH
 
+    @staticmethod
+    def get_run_dir_by_sequencer_type(
+        config: dict,
+        sequencer_type: Optional[str] = None,
```

Comment:
I don't think `sequencer_type` should be optional (since that causes a key error below)